lms/templates/fields/field_dropdown.underscore

```diff
@@ -17,11 +23,11 @@
             <option value="<%- selectOption[0] %>"><%- selectOption[1] %></option>
         <% }); %>
     </select>
-    <a href="#" class="u-field-value-display">
-        <span class="sr"><%- screenReaderTitle %></span>
+    <button class="u-field-value-display">
+        <span class="sr"><%- screenReaderTitle %> &nbsp;</span>
```

Comment:
@ahsan-ul-haq @cptvitamin The `screenReaderTitle` is already being added here. Do we want two instances of it in the field?